styles.css

```diff
@@ -4,289 +4,372 @@
+    font-size: 18px;
+    background-color: white;
+    border: 1px #396df2 solid;
+    border-radius: 8px;
+    color: #396df2;
+    cursor: pointer;
+    margin-top: 20px;
+  }
+
+  .button:hover {
+    color: black;
+    background-color: rgb(236 127 48);
+    border-color: chocolate;
+  }
+
+  .portfolio-two {
+    background-color: #e5e5e5;
+    margin: 0 auto;
+    width: 100%;
+    position: static;
+    padding-top: 60px;
+  }
+
+  .portfolio-card {
+    display: grid;
+    background-color: #e5e5e5;
+    padding-left: 8px;
+    padding-top: 10px;
+    padding-right: 15px;
+    width: 375px;
+    margin: 0 auto;
+  }
+
+  .portfolio-card-about {
+    padding-bottom: 40px;
+    width: 100%;
+    border-top-right-radius: 60px;
+  }
+
+  .portfolio {
+    padding: 20px;
+    background-color: white;
+    display: grid;
+    align-items: center;
+    align-content: center;
+    border-radius: 16px;
+    margin-bottom: 90px;
+    position: sticky;
+    width: 375px;
+  }
+
+  .portfolio-card-about .portfolio {
+    width: 100%;
+    border-top-right-radius: 80px;
+  }
+
+  .portfolio-images img {
+    width: 320px;
+    height: 220px;
+    margin: 0 auto;
+    margin-top: 5px;
+    margin-bottom: 30px;
+  }
+
+  .porfolio-flex {
+    display: flex;
+    align-content: center;
+    align-items: baseline;
+  }
+
+  .porfolio-flex p {
+    margin-right: 30px;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #344563;
+  }
+
+  .portfolio-paragraph {
+    margin-top: 10px;
+    margin-left: 10px;
+  }
+
+  .portfolio-paragraph h2 {
+    font-size: 36px;
+    margin-bottom: 15px;
+  }
+
+  .portfolio-paragraph-canopy {
+    display: flex;
+    align-items: baseline;
+    width: 300px;
+    margin-top: 10px;
+  }
+
+  h1 {
+    font-size: 36px;
+    margin-top: -20px;
+  }
+
+  .btn-tonic {
+    padding: 10px;
+    width: 140px;
+    font-size: 18px;
+    background-color: white;
+    border: 1px #396df2 solid;
+    border-radius: 8px;
+    color: #396df2;
+    cursor: pointer;
+    margin-top: 20px;
+  }
+
+  .btn-tonic:hover {
+    background-color: #396df2;
+    color: white;
+    border: 1px solid white;
+  }
+
+  .porfolio-courses ul li {
+    margin-right: 10px;
+  }
+
+  .porfolio-courses ul li a {
+    margin-top: 20px;
+    text-decoration: none;
+    color: #6070ff;
+    background-color: #ebebff;
+    padding: 4px 12px;
+    border-radius: 8px;
+  }
+
+  .portfolio-paragraph h3 {
+    font-size: 30px;
+  }
+
+  .portfolio-paragraph h4 {
+    font-size: 32px;
+  }
+
+  .images-languages {
+    margin-top: 10px;
+    margin-left: 10px;
+    display: block;
+  }
+
+  .images-languages img {
+    margin-bottom: 10px;
+  }
+
+  .about-card {
+    margin-top: 20px;
+  }
+
+  .about-card ul {
+    display: flex;
+    flex-direction: column;
+    list-style: none;
+  }
+
+  .images-languages-two {
+    display: none;
+  }
+
+  .about-card .languages {
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] The about section of the desktop version still has some issues. The width of the section doesn't fit the screen and the whole page is affected by this issue. I took a screenshot of the issue. It would be nice to fix it to make your desktop version design better.
<img width="564" alt="image" src="https://user-images.githubusercontent.com/77190203/150568317-6a121947-6658-496b-a74f-2cbbe79dcdda.png">